assets/js/modules/analytics/datastore/__fixtures__/create-account.json

```diff
@@ -0,0 +1,6 @@
+{
+  "accountName": "Test Account",
+  "propertyName": "Test Property",
+  "profileName": "Test Profile",
+  "timezone": "Test Timezone."
+}
```

Comment:
Would be great to use a fixture here that looks like an actual response from our API endpoint (account ticket object with nested objects for accounts etc). We should at least make that change as we start testing against the real API - it'll be easy then to record a real response and place some dummy values in there.

Reply:
updated fixture